_includes/codepens/editor-dfree/index.js

```diff
@@ -21,12 +21,14 @@ var dfreeBodyConfig = {
     'powerpaste',
     'table',
     'image',
-    'quickbars'
+    'quickbars',
+    'codesample',
+    'help'
   ],
   toolbar: false,
-  quickbars_insert_toolbar: 'quicktable image',
-  quickbars_selection_toolbar: 'bold italic | h2 h3 | blockquote quicklink',
-  contextmenu: 'inserttable | cell row column deletetable',
+  quickbars_insert_toolbar: 'quicktable image media codesample',
+  quickbars_selection_toolbar: 'bold italic underline | h2 h3 | blockquote quicklink',
+  contextmenu: 'formats | inserttable | cell row column deletetable | help',
```

Comment:
I can't say I really like the idea of including formats in a context menu... maybe put it in the selection toolbar instead?

Reply:
Move format select to the toolbar, in-place of h2 and h3.
Added undo and redo to the context menu